src/components/store/mypage/ReservationList.js

```diff
@@ -19,62 +21,64 @@ const ReservationList = ({ reservations }) => {
         });
     }, []);
 
+    const handleReservationClick = (reservation) => {
+        openModal('storeReservationDetail', { reservation });
+    };
+
```

Comment:
이렇게 수정하면 예약 상세 내역 모달이 정상 작동됩니다!

```suggestion
    const handleReservationClick = (reservation) => {
        openModal('storeReservationDetail', { reservationInfo: reservation });
    };

```

Reply:
넵